Paco-Server/src/com/google/sampling/experiential/server/EventRetriever.java

```diff
@@ -267,7 +268,12 @@ public void sendToCloudSqlQueue(JSONObject eventJson, Event event) {
     } catch (JSONException e) {
       log.severe("while sending to cloud sql queue" + e);
     }
-    queue.add(TaskOptions.Builder.withUrl("/csInsert").payload(eventJson.toString()));
+    TaskOptions to = TaskOptions.Builder.withUrl("/csInsert").payload(eventJson.toString());
+    if (!System.getProperty("com.google.appengine.runtime.version").startsWith("Google App Engine/")) {
```

Comment:
As discussed, let's make a util class with a method, isDevelopmentServer, to check this.